### PR TITLE
Fixed issue with scratch FGDB path in Configuration

### PR DESCRIPTION
**Pull request by mfunk**
Was set to NONE in configuation and assumed tests would create, but this caused more errors than it tried to solve. Now defaulting to a relative path in the .\utils\test folder.

### CODE REVIEW

**Comment by mfunk** [comment]
@topowright just a request to check this one out and see if it makes sense.

**Comment by mfunk** [comment]
@csmoore any chance you want to take a crack at this one?

File: utils/test/Configuration.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 militaryDataPath = os.path.normpath(os.path.join(currentPath, r"../../testdata/"))
 militaryInputDataGDB = os.path.normpath(os.path.join(militaryDataPath, "MilitaryToolsTestData.gdb"))
 militaryResultsGDB = os.path.normpath(os.path.join(militaryDataPath, "Results.gdb"))
-militaryScratchGDB = None
+militaryScratchGDB = os.path.normpath(os.path.join(currentPath, "scratch.gdb"))
```
Comment (csmoore) [inline utils/test/Configuration.py]: Do you intend this to go in the folder: `military-tools-geoprocessing-toolbox\utils\test\scratch.gdb` - if so you may want to add a gitignore to that folder to ignore this.

**Comment by csmoore** [comment]
I tried to run, but am getting that odd error where the testsuite tries to read the toolbox file:

```
dding test: test_linear_line_of_sight_pro
adding test: test_radial_line_of_sight_pro
adding test: test_add_llos_fields_pro
adding test: test_add_rlos_observer_fields_pro
running 68 tests...
.....ConvertCoordinatesTestCase.test_convert_coordinates_pro
ERROR: Start Time: Tuesday, April 4, 2017 12:01:53 PM
ERROR 000210: Cannot create output C:\MyFiles\Work\Github\military-tools-geoprocessing-toolbox\utils\test\scratch.gdb\outputConvert
Failed to execute (CopyRows).
Failed at Tuesday, April 4, 2017 12:01:53 PM (Elapsed Time: 0.12 seconds)
Start Time: Tuesday, April 4, 2017 12:01:53 PM
ERROR 000210: Cannot create output C:\MyFiles\Work\Github\military-tools-geoprocessing-toolbox\utils\test\scratch.gdb\outputConvert
Failed to execute (CopyRows).
Failed at Tuesday, April 4, 2017 12:01:53 PM (Elapsed Time: 0.12 seconds)
Traceback (most recent call last):
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\lib\tokenize.py", line 392, in find_cookie
    line_string = line.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd0 in position 0: invalid continuation byte
...
SyntaxError: invalid or missing encoding declaration for 'C:\\MyFiles\\Work\\Github\\military-tools-geoprocessing-toolbox\\toolboxes\\Military_Tools_pro.tbx'
'One or more tests failed'
```

I can't remember how this was resolved last time.

**Comment by mfunk** [comment]
We didn't. It was submitted as a bug with core. I don't remember the bug number. Will have to look it up.

**Comment by mfunk** [comment]
BUG-000099597 : SyntaxError: invalid or missing encoding declaration for ArcGIS Pro toolbox
This one is still Open in their queue.

**Comment by csmoore** [comment]
I worked around the [Pro BUG-000099597 issue](https://geonet.esri.com/thread/179895#comment-678404)

But when I run both the ArcMap and Pro tests about half of them fail.  Logs: [MTGT_Logs.zip](https://github.com/Esri/military-tools-geoprocessing-toolbox/files/906509/MTGT_Logs.zip)

I doubt these failures are related the this particular change, so just let me know if I should just merge. I'll also submit a PR to this branch with the changes I made in order to get to run.

**Comment by mfunk** [comment]
.... and are we back to re-review this one or OK with fixes from other #213 PR?

**Pull request merged**